src/components/editor/property/InputList.jsx

```diff
@@ -109,6 +110,7 @@ InputList.propTypes = {
   propertyTemplate: PropTypes.object.isRequired,
   values: PropTypes.array,
   addValue: PropTypes.func,
+  propertyLabelId: PropTypes.string.isRequired,
```

Comment:
i made this a required prop in all of the places where it seemed like the value would always be provided (by `grep`ing for usage of the component).  happy to loosen if this seems overly strict.

Reply:
That is fine, I prefer having props to be strict as it usually points out an error or edge case we are not handling properly.

Reply:
cool, that was also my thinking